en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,250 @@
 # Create Your First Service
 
-To learn how to create a service, let's look at a simple scenario. In this example, you'll create and design a service that consumes a public API that fetches the current active COVID-19 cases in the United States. Then you'll test it, deploy it, and observe its performance.
+This quick start guide walks you through the steps to create a service that gets the current COVID-19 status of a given country. It will:
 
-## Step 1: Create the service
+- Connect to the COVID-19 data API to get COVID-19 statistics.
+- Connect to the world bank data API to get population data.
+- Send a response back with further processed COVID-19 data.
 
-To create the service, follow this procedure:
+Once you develop and test the service, you can deploy it and observe its statistics to evaluate its performance.
 
-1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
-    
-2. On the **Services** page, click **Create**. You are directed to the **Create Service** page.
+## Step 1: Create the service resources
 
-3. Under **Create with Choreo**, enter `covid-stats` as the name of your service, and then click **Create**. You are directed to the low-code view to design the service.
+To create the service resources:
 
-## Step 2: Design the service
+1. Access the Choreo Console via [https://console.choreo.dev/](https://console.choreo.dev/).
+    
+2. Sign in using either your Google or GitHub credentials.
+    
+3. On the **Services** page, click **Create**. You are directed to the low-code view.
 
-To design the service to fetch the current active COVID-19 cases in the United States, follow this procedure:
+4. Under **Create with Choreo**, enter `CovidStatus` as the name of your service.
 
-1. Select **GET** as the HTTP method and enter `activecases` as the relative path from the host.
+5. Select **GET** as the HTTP method, and enter `stats/[string country]/[string date]` in the **Path** field.
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png){.cInlineImage-half}
+    ![Resource configuration](../assets/img/services/configure-api-trigger.png){.cInlineImage-half}
```

Comment:
This image does not show the correct path.

Reply:
Updated